machine-learning/Dockerfile

```diff
@@ -1,34 +1,27 @@
-FROM node:16-bullseye-slim as builder
-
-ARG DEBIAN_FRONTEND=noninteractive
+# first stage
+FROM python:3.10 AS builder
 
 WORKDIR /usr/src/app
 
-RUN apt-get update
-RUN apt-get install gcc g++ make cmake python3 python3-pip -y
-
-COPY package.json package-lock.json ./
-
-RUN npm ci
-RUN npm rebuild @tensorflow/tfjs-node --build-from-source
-
+# COPY requirements.txt .
 COPY . .
```

Comment:
Can this step be moved to after the dependency installation for proper layering? (And, if we stick with the multi-stage build, it doesn't need to be in this stage at all)